include/libhal-sensor/multi/bmp180.hpp

```diff
@@ -0,0 +1,123 @@
+// Copyright 2024 Khalil Estell
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <cstdint>
+
+#include <libhal/i2c.hpp>
+#include <libhal/units.hpp>
+
+namespace hal::sensor {
+
+/**
+ * @brief Driver for the BMP180 barometric pressure/temperature sensor
+ */
+
+class bmp180
+{
+public:
+  /**
+   * @brief Oversampling rate defined with the mode names as well as sample
+   * fetch times.
+   */
+  enum class oversampling_rate : hal::byte
+  {
+    /// 3 internal samples by sensor
+    ulta_low_power_mode_4500us = 0x00,
+    /// 5 internal samples by sensor
+    standard_mode_7500us = 0x01,
+    /// 9 internal samples by sensor
+    high_resolution_mode_13500us = 0x02,
+    /// 17 internal samples by sensor
+    ultra_high_resolution_mode_25500us = 0x03,
+  };
+
+  /**
+   * @brief Stores the temperature and pressure data that are both found when
+   * doing the pressure()
+   */
+  struct pressure_results
+  {
+    hal::celsius temperature;
+    float pressure;
```

Comment:
```suggestion
    // Pressure is in units of "____"
    float pressure;
```

Please add the units for pressure for future usage.